README.md

```diff
@@ -181,6 +181,34 @@ See the [Mailchimp GDPR Tutorial](https://mailchimp.com/help/collect-consent-wit
 #### Retrieving Marketing Permission Keys
 Coming Soon
 
+## Collecting contact events
+
+### Adding an event
+
+To add an event associated with a contact, pass the email, event name and properties (optional) into the `addContactEvent` method. This will add the event to the specified contact.
```

Comment:
Think we might want an oxford comma here. Otherwise I'm confused if the "(option)" applies to just "properties" or both "event name and properties".

Reply:
Good point. Fixed.